packages/components/psammead-script-switch/README.md

```diff
@@ -0,0 +1,62 @@
+# psammead-script-switch - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-script-switch%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-script-switch%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-script-switch)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-script-switch) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-script-switch)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-script-switch&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/script-switch--containing-image) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-script-switch.svg)](https://www.npmjs.com/package/@bbc/psammead-script-switch) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `SwitchScript` component is designed to be used where a service has multiple variants and there is a need to switch between them.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-script-switch --save
+```
+
+## Props
+
+<!-- prettier-ignore -->
+| Argument  | Type | Required | Default | Example |
+| --------- | ---- | -------- | ------- | ------- |
+| service | string | yes | N/A | `'serbian'` |
+| variant | string | no | `null` | `'lat'` |
+| href | string | yes | N/A | `'https://www.bbc.com/serbian/lat'` |
+| children | node | yes | N/A | `'Lat'` |
+| script | object | yes | N/A | { canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, } |
+
+## Usage
+
+```jsx
+import ScriptSwitch from '@bbc/psammead-script-switch';
+import { latin } from '@bbc/gel-foundations/scripts';
+
+const WrappingComponent = () => (
+  <div>
+    <ScriptSwitch
+      script={latin}
+      service="serbian"
+      href="https://www.bbc.com/serbian/lat"
+      variant="lat"
+    >
+      Lat
+    </ScriptSwitch>
+  </div>
+);
+```
+
+### When to use this component
+
+The `SwitchScript` component is to be used on the `Brand` component.
+
+### Accessibility notes
+
+<!-- Information about accessibility for this component -->
```

Comment:
`This component is still in its initial alpha stages, and requires a full and extensive accessibility review.`